pages/validator/external-chains/linea.mdx

```diff
@@ -118,6 +118,7 @@ Create the systemd service file `/etc/systemd/system/geth-linea.service`
 </Callout>
 
 ```ini
+sudo tee <<EOF >/dev/null /etc/systemd/system/geth-linea.service
```

Comment:
Please consider to use following command instead, pretty much same thing but with minor adjustment
`sudo tee /etc/systemd/system/geth-linea.service &>/dev/null <<EOF`